interactions/client/get.py

```diff
@@ -0,0 +1,394 @@
+        gotten.
+
+        * Example 1/2: Getting a Channel:
+
+            .. code-block:: python
+
+                # normally
+                channel = await get(client, interactions.Channel, object_id=your_channel_id)
+                # always has a value
+
+                # with http force
+                channel = await get(client, interactions.Channel, object_id=your_channel_id, force="http")
+                # always has a value
+
+                # with cache force
+                channel = get(client, interactions.Channel, object_id=your_channel_id, force="cache")
+                # because of cache only, this can be None
+
+
+        * Example 2/2: Getting a Member:
+
+            .. code-block:: python
+
+                # normally
+                member = await get(
+                    client, interactions.Member, parent_id=your_guild_id, object_id=your_member_id
+                )
+                # always has a value
+
+                # with http force
+                member = await get(
+                    client, interactions.Member, parent_id=your_guild_id, object_id=your_member_id
+                )
+                # always has a value
+
+                # with cache force
+                member = await get(
+                    client, interactions.Member, parent_id=your_guild_id, object_id=your_member_id
+                )
+                # because of cache only, this can be None
+
+
+        Both examples should have given you a basic overview on how to get a single object. Now we will move on with
+        lists of objects.
+
+        .. important::
+            The ``parent_id`` represents the channel or guild id that belongs to the objects you want to get. It is
+            called ``parent_id`` bcause ``guild_or_channel_id`` would be horrible to type out every time.
+
+    Getting a list of an object:
+        Here you will see 1 example of how to get a list of objects. The possibilities on how to force (and their
+        results) are the same as in the examples above.
+
+        * Example 1/1: Getting a list of members:
+
+            .. code-block:: python
+
+                from typing import List
+
+                # you can also use `list[interactions.Member]` if you have python >= 3.9
+
+                members = await get(
+                    client,
+                    List[interactions.Member],
+                    parent_id=your_guild_id,
+                    object_ids=[your_member_id1, your_member_id2, ...],
+                )
+
+
+        If you enforce cache when getting a list of objects, found objets will be placed into the list and not found
+        objects will be placed as ``None`` into the list.
+
+
+
+    """
+
+    if version_info >= (3, 9):
+
+        def _check():
+            return obj == list[get_args(obj)[0]]
+
+    else:
+
+        def _check():
+            return False
+
+    if len(args) == 2 and any(isinstance(_, Iterable) for _ in args):
```

Comment:
Condense these two conditionals which share the same comparison on the main scope.

Reply:
Wdym

Reply:
Did smth maybe you meant that idk